packages/graphics/libmali/package.mk

```diff
@@ -2,26 +2,36 @@
 # Copyright (C) 2019-present Team LibreELEC (https://libreelec.tv)
 
 PKG_NAME="libmali"
-PKG_VERSION="ad56ed30985471c0950a654cc9db1e86310650d5"
-PKG_SHA256="72438ea73cf6c2e8e770545872386b66ccad200c568261e5304845547099c9ed"
+PKG_VERSION="4cbf211cfd9b07854aab4978e50b1151052c6d4c"
+PKG_SHA256="d3c5dd43ee0830feada4a86d32b9c794cddab29a863dfd1dddb3954f380086c5"
 PKG_ARCH="arm aarch64"
 PKG_LICENSE="nonfree"
 PKG_SITE="https://github.com/LibreELEC/libmali"
 PKG_URL="https://github.com/LibreELEC/libmali/archive/$PKG_VERSION.tar.gz"
 PKG_LONGDESC="OpenGL ES user-space binary for the ARM Mali GPU family"
+PKG_STAMP="$MALI_FAMILY"
 
 PKG_DEPENDS_TARGET="libdrm"
 
-if [ "$MALI_FAMILY" = "t620" -o "$MALI_FAMILY" = "t720" -o "$MALI_FAMILY" = "g52" ]; then
+if [ "$MALI_FAMILY" = "t620" -o "$MALI_FAMILY" = "t720" ]; then
```

Comment:
This also needs to be changed if `t620` or `t720` is to be included in the image, e.g.
```
case $MALI_FAMILY in
  *t620*|*t720*)
```

Reply:
You could also consider using `listcontains` for multiple "or" choices, as we do here:

https://github.com/LibreELEC/LibreELEC.tv/blob/e13ed3899f8cdf3cd90f52dd4d51ec4d0ed0c056/packages/graphics/libdrm/package.mk#L23

ie.
```
if listcontains "${MALI_FAMILY}" "(t620|t720)"; then
  PKG_DEPENDS_TARGET+=" wayland"
fi
```

Reply:
Thanks, I will fix this